mvc-web/src/main/kotlin/com/example/model/DatabaseFactory.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.model
+
+import com.example.model.dao.wishlist.Wishlists
+import kotlinx.coroutines.Dispatchers
+import org.jetbrains.exposed.sql.Database
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.transactions.experimental.newSuspendedTransaction
+import org.jetbrains.exposed.sql.transactions.transaction
+
+object DatabaseFactory {
```

Comment:
In other examples, we renamed "DatabaseFactory" to "DatabaseSingleton". I think we should keep the names consistent.